src/components/RadioTile/RadioTile-test.js

```diff
@@ -64,6 +64,10 @@ describe('RadioButton', () => {
         wrapper.setProps({ labelText: 'test label text' });
         expect(label.text()).toMatch(/test label text/);
       });
+
+      it('should not have the className when no class is passed through props', () => {
```

Comment:
Could you add a test for a custom class name as well? Thanks! 🙌

Reply:
will do 👍